src/features/conversations/components/ConversationPanel.module.css

```
@@ -8,6 +8,8 @@
   letter-spacing: 0.02em;
   font-weight: 350;
   font-size: 1em;
+
+  text-shadow: var(--text-shadow-subtle);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)